### PR TITLE
[#8973] Mobile data syncing pop-up missing text

### LINKED ISSUE #8973: Mobile data syncing pop-up missing text
# Problem

On the nightly, the 'sync mobile data' modal shows only the icons for each option, with no text:

## Expected behavior

Text describes each option:

`Continue syncing`
`Stop syncing`

### PR DESCRIPTION
**Pull request by flexsurfer**
fixes #8973

image

tested in ios simulator

### CODE REVIEW

**Comment by ghost** [comment]
### Pull Request Checklist
- [ ] **Docs**: Updated the [documentation](https://status.im/build_status/), if affected
- [ ] **Docs**: Added or updated inline comments explaining intention of the code
- [ ] **Tests**: Ensured that all new UI elements have been assigned accessibility IDs
- [ ] **Tests**: Signaled need for E2E tests with label, if applicable
- [x] **Tests**: Briefly described what was tested and what platforms were used
- [x] **UI**: In case of UI changes, ensured that UI matches [Figma](https://www.figma.com/)
- [ ] **UI**: In case of UI changes, requested review from a Core UI designer
- [x] **UI**: In case of UI changes, included screenshots of implementation

**Review by vitvly — APPROVED** [review]

File: src/status_im/ui/screens/mobile_network_settings/sheets.cljs [hunk 1]
```diff
@@ -56,8 +56,9 @@
 
 (views/defview settings-sheet []
   [react/view {:style styles/container}
```
Comment (yenda) [inline src/status_im/ui/screens/mobile_network_settings/sheets.cljs]:
```suggestion
  [react/view {:style {:flex 1}
```

File: src/status_im/ui/screens/mobile_network_settings/sheets_styles.cljs [hunk 2]
```diff
@@ -1,10 +1,8 @@
 (ns status-im.ui.screens.mobile-network-settings.sheets-styles
-  (:require [status-im.ui.components.colors :as colors]
-            [status-im.ui.components.styles :as common-styles]))
+  (:require [status-im.ui.components.colors :as colors]))
 
 (def container
-  {:flex        1
-   :align-items :center})
+  {:flex        1})
```
Comment (yenda) [inline src/status_im/ui/screens/mobile_network_settings/sheets_styles.cljs]:
```suggestion
```

File: src/status_im/ui/screens/mobile_network_settings/sheets_styles.cljs [hunk 3]
```diff
@@ -1,10 +1,8 @@
 (ns status-im.ui.screens.mobile-network-settings.sheets-styles
-  (:require [status-im.ui.components.colors :as colors]
-            [status-im.ui.components.styles :as common-styles]))
+  (:require [status-im.ui.components.colors :as colors]))
 
 (def container
```
Comment (yenda) [inline src/status_im/ui/screens/mobile_network_settings/sheets_styles.cljs]:
```suggestion
```

**Comment by yenda** [comment]
I tested it works but lets remove the useless style

**Pull request merged**